mycore-restapi/src/main/java/org/mycore/restapi/v2/MCREventHandler.java

```diff
@@ -287,7 +287,7 @@ public void sendEvent(MCREvent evt, Path path, BasicFileAttributes attrs) {
             file.addProperty("object", objId);
             file.addProperty("size", attrs.size());
             file.addProperty("modified", attrs.lastModifiedTime().toInstant().toString());
-            file.addProperty("md5", ((MCRFileAttributes) attrs).md5sum());
+            file.addProperty("md5", ((MCRFileAttributes) attrs).digest().getValue());
```

Comment:
I think its ok to deliver a sha512 hash as md5, because the api would be more stable for old applications. Alternative would be using the digest algo name as key.